include/deal.II/hp/dof_handler.h

```diff
@@ -627,6 +626,23 @@ namespace hp
     n_locally_owned_dofs_per_processor () const;
 
     /**
+     * Return an IndexSet describing the set of locally owned DoFs used for the
+     * given multigrid level. Since hp::DoFHandler does not support multilevel
+     * methods yet, this function returns an empty index set, IndexSet(0),
```

Comment:
do you want to adjust docu to: "... this function throws an error if called"? same below.